src/Sets/Zonotope.jl

```diff
@@ -427,3 +427,56 @@ function linear_map!(Zout::Zonotope, M::AbstractMatrix, Z::Zonotope)
     mul!(Zout.generators, M, Z.generators)
     return Zout
 end
+
+"""
+    quadratic_map(Q::Vector{MT}, Z::Zonotope{N}) where {N, MT<:AbstractMatrix{N}}
+
+Return the overapproximation of the quadratic map of the given zonotope.
```

Comment:
@SebastianGuadalupe  can you add the mathematical definition here? which will show in paricular that each `Q[i]` is used for each coordinate of the solution.